app/design/adminhtml/default/default/template/eav/attribute/options.phtml

```diff
@@ -166,6 +186,27 @@ var attributeOption = {
             this.updateItemsCountField();
         }
     },
+    swatch : function(event){
+        var element = $(Event.findElement(event, 'tr')); // !!! Button already
+                                                               // have table parent in safari
+        // Safari workaround
+        element.ancestors().each(function(parentItem){
```

Comment:
thi is also probably prototypejs

Reply:
ok but each could be simply replaced by forEach right? which would probably be better anyway no?

Reply:
sure i've unlocked editing, I am grateful if you contribute